server/fishtest/rundb.py

```diff
@@ -603,15 +603,22 @@ def calc_itp(self, run):
             itp = 1
         elif itp > 500:
             itp = 500
-        itp *= math.sqrt(
-            estimate_game_duration(run["args"]["tc"]) / estimate_game_duration("10+0.1")
-        )
-        itp *= math.sqrt(run["args"]["threads"])
-        if "sprt" not in run["args"]:
-            itp *= 0.5
-        else:
+
+        # Base TP derived from power law of TC relative to STC
+        tc_ratio = run["args"]["threads"] * estimate_game_duration(run["args"]["tc"]) \
```

Comment:
We use black to format the code form time to time, feel free to keep long lines.

Reply:
I've always thought that autoformatters tend to produce suboptimal results. If it's going to be spread over two lines, I may as well make that look the best I can, and line up the two function calls

Reply:
I routinely run black, isort, prettier on the code base, so the format by hand could be changed.